civihr_default_theme/scss/base/_variables.scss

```diff
@@ -1,5 +1,9 @@
 @import url('//fonts.googleapis.com/css?family=Lato:400,300,300italic,400italic,700,700italic');
 
+
+$chr-gradient-1: linear-gradient(315deg, #01A2D5 0%, #0070BB 100%);
+$chr-gradient-2: linear-gradient(315deg, #0187B1 0%, #005B97 100%);
+
```

Comment:
i believe these gradients should be place right after the colors list, given that they're still colors